DatadogSessionReplay/Tests/Recorder/ViewTreeSnapshotProducer/ViewTreeSnapshot/NodeRecorders/UIImageViewRecorderTests.swift

```diff
@@ -81,6 +81,62 @@ class UIImageViewRecorderTests: XCTestCase {
         XCTAssertNotNil(builder.imageResource)
     }
 
+    func testWhenMaskAllImagePrivacy_itDoesNotRecordImage() throws {
+        // Given
+        let imagePrivacy = ImagePrivacyLevel.maskAll
+        let context = ViewTreeRecordingContext.mockWith(recorder: .mockWith(imagePrivacy: imagePrivacy))
+
+        // When
+        let recorder = UIImageViewRecorder()
+        imageView.image = UIImage()
+        viewAttributes = .mock(fixture: .visible())
+
+        // Then
+        let semantics = try XCTUnwrap(recorder.semantics(of: imageView, with: viewAttributes, in: context))
+        XCTAssertTrue(semantics is SpecificElement)
+        XCTAssertEqual(semantics.subtreeStrategy, .record, "Image view's subtree should be recorded")
+        let builder = try XCTUnwrap(semantics.nodes.first?.wireframesBuilder as? UIImageViewWireframesBuilder)
+        XCTAssertNil(builder.imageResource)
+    }
+
+    func testWhenMaskNoneImagePrivacy_itDoesRecordImage() throws {
+        // Given
+        let imagePrivacy = ImagePrivacyLevel.maskNone
+        let context = ViewTreeRecordingContext.mockWith(recorder: .mockWith(imagePrivacy: imagePrivacy))
+
+        // When
+        let recorder = UIImageViewRecorder()
+        imageView.image = UIImage()
+        viewAttributes = .mock(fixture: .visible())
+
+        // Then
+        let semantics = try XCTUnwrap(recorder.semantics(of: imageView, with: viewAttributes, in: context))
+        XCTAssertTrue(semantics is SpecificElement)
+        XCTAssertEqual(semantics.subtreeStrategy, .record, "Image view's subtree should be recorded")
+        let builder = try XCTUnwrap(semantics.nodes.first?.wireframesBuilder as? UIImageViewWireframesBuilder)
+        XCTAssertNotNil(builder.imageResource)
+    }
+
+    func testWhenMaskContentImagePrivacy_itDoesRecordSFSymbolImage() throws {
+        // Given
+        let imagePrivacy = ImagePrivacyLevel.maskContent
+        let context = ViewTreeRecordingContext.mockWith(recorder: .mockWith(imagePrivacy: imagePrivacy))
+
+        // When
+        let recorder = UIImageViewRecorder()
+        if #available(iOS 13.0, *) {
```

Comment:
Should we add a comment here to explain why we only set the image for iOS 13+?

Reply:
Technically speaking SR works only for iOS 13+, but SPM does not give us granularity over supported systems versions for each target.